src/System.CommandLine.ApiCompatibility.Tests/ApiCompatibilityApprovalTests.System_CommandLine_api_is_not_changed.approved.txt

```diff
@@ -434,6 +436,7 @@ System.CommandLine.Parsing
     public System.Object GetValue(System.CommandLine.Argument argument)
     public T GetValue<T>(Option<T> option)
     public System.Object GetValue(System.CommandLine.Option option)
+    public System.Void ReportError(System.String errorMessage)
```

Comment:
I'd suggest `AddError` since nothing is actually being reported at this point.

Reply:
@jonsequitur done